aws/velero-bucket/outputs.tf

```diff
@@ -1,17 +1,19 @@
 output "bucket_name" {
-  value = element(concat(aws_s3_bucket.this.*.id, list("")), 0)
+  value = try(aws_s3_bucket.this.*.id[0], "")
 }
 
 output "velero_iam_user_name" {
-  value = element(concat(aws_iam_user.velero_iam_user.*.name, list("")), 0)
+  value = try(aws_iam_user.velero_iam_user.*.name[0], "")
 }
 
 output "velero_iam_user_access_key" {
-  value = element(concat(aws_iam_access_key.velero_iam_access_key.*.id, list("")), 0)
+  value     = try(aws_iam_access_key.velero_iam_access_key.*.id[0], "")
+  sensitive = true
```

Comment:
The key ID isn't a secret, but I  guess it doesn't hurt much either to mark it as one.